order_guidelines.md

```diff
@@ -77,5 +77,4 @@ The `Tags` Element is the one that is most subject to changes after the file was
 
 ## Cluster Timecode
 
-As each `BlockGroup` and `SimpleBlock` of a `Cluster` Element needs the Cluster `Timecode`, the `Timecode` Element MUST occur as the first Child Element within the `Cluster` Element.
-
+As each `BlockGroup` and `SimpleBlock` of a `Cluster Element` needs the Cluster `Timecode Element`, the `Timecode Element` MUST occur as the first `Child Element` within the `Cluster Element`.
```

Comment:
Could simplify Cluster `Timecode Element` to just `Timecode Element`.
Also there is a conflict here, when `Cluster` has a `CRC-32` I would expect it to be first, but this mandates that `Timecode` is first, thus with rules as written it is not possible to have a valid `Cluster Element` with a CRC.